src/court_spec/schemas/courtSpec-example.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Pro Full Court",
+    "length": 28000,
+    "width": 15000,
+    "centreCircleRadius": 1800,
+    "threePointRadius": 6750,
+    "threePointLine": 900,
+    "lengthOfCorner": 2990,
+    "restrictedAreaLength": 5800,
+    "restrictedAreaWidth": 4900,
+    "sideBorderWidth": 2000,
+    "lineBorderWidth": 50,
+    "description": "This is the default PRO Full Court"
+}
```

Comment:
Do we really need this example? I'm just curious. I will add one in my code if we need this.

Reply:
I think it is for reference.